_data/accessibility-tests/header.yml

```diff
@@ -0,0 +1,243 @@
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+  - summary: Header is at least partially visible when focus moves across other interactive elements. 
+    summary_additional: |
+      When you use a keyboard and tab across elements in a header, 
+      focus on interactive elements may overlay the header, but header will always appear partially visible in background. 
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.11
+  - summary: Focus indicator is clearly visible and consistent.
+    summary_additional: |
+      When you use a keyboard to navigate through the header, 
+      the focus indicator has the same sizing, contrast and perimeter, regardless of the element selected.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.13
+# Screen reader tests
+  - summary:
+    summary_additional:
+    test_status:
+    test_type: screen_reader
+    version_tested:
+    wcag_criterion:
+  - summary: Screen reader announces header as a navigation landmark.
+    summary_additional: |
+      When you use a screen reader and access the header, 
+      "navigation region" is announced. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces header elements in logical order.
+    summary_additional: |
+      When you use a screen reader and reach the header, 
+      elements will be announced in the order they appear visually.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Magnifying glass icon in the header is only used to indicate "search" in every instance on the site.
+    summary_additional: |
+      When you see or tab to the magnifying glass icon in any header, 
+      it will always and only be used to indicate search.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.4
+  - summary: Screen reader announces role values.
+    summary_additional: |
+      When you use a screen reader and access a button or text input option, 
+      you hear that the value of the role. 
+      For example: "button", "text edit".
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 4.1.2
+  - summary: Screen reader announces current page.
+    summary_additional: | 
+      When landing on any page in the header navigation, 
+      the role of "current" is announced.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 4.1.2
+  - summary: Screen reader announces search field purpose.
+    summary_additional: |
+      When you use a screen reader and access a search field in a header, 
+      ou can hear that you are able to edit what is in the search field. 
```

Comment:
@RachelCorsino 
**issue:** Typo - missing "y"in "you"
```suggestion
      you can hear that you are able to edit what is in the search field. 
```

Reply:
Resolved in [2aae0a9](https://github.com/uswds/uswds-site/pull/2761/commits/2aae0a950202ec0f7b7e202b7ac13cef50ff2895)